### PR TITLE
feat: add category filter on talk overview page

### PR DESCRIPTION
**Pull request by yuyanghh**
## Types of changes

* **New feature**

## Description

Apply the category filter to display the cards with user-selected  categories on talk overview page
As presets, all categories are unselected and all cards are displayed.

https://user-images.githubusercontent.com/13831865/130360683-dd9365c5-9a2c-4526-b89d-c10341b2b1cc.mov

## Related Issue

#134

### CODE REVIEW

File: components/core/checkbox/Checkbox.vue [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    },
+    computed: {
+        isChecked() {
+            if (typeof this.value === 'boolean') {
+                return this.value
+            } else {
+                return this.value.includes(this.id)
+            }
+        },
+    },
```
Comment (mattwang44) [inline components/core/checkbox/Checkbox.vue:62]: How about directly passing `isChecked` props to this component? Defining whether a checkbox should be checked in the caller of the checkbox component would be better.

File: pages/conference/_eventType/index.vue [hunk 2]
```diff
@@ -4,14 +4,39 @@
             <core-h1 :title="title"></core-h1>
             <p class="intro">{{ intro }}</p>
         </i18n-page-wrapper>
+        <i18n-page-wrapper
+            v-if="eventType === 'talks'"
+            class="px-4 sm:px-8 md:px-16 lg:px-32 pb-6 lg:pb-12"
+            custom-x
+            custom-y
+        >
+            <p class="filter ml-2">Filter</p>
```
Comment (mattwang44) [inline pages/conference/_eventType/index.vue:13]: - Change the class name would be more comprehensible. 
- Move `ml-2` to css block.

```suggestion
            <p class="filterTitle">Filter</p>
```

**Review by mattwang44 — CHANGES_REQUESTED** [review]
It works perfectly 👍. Commented suggestions are for better readability and more comprehensible.

File: components/core/checkbox/Checkbox.vue (hunk 1 above)
Comment (yuyanghh) [reply components/core/checkbox/Checkbox.vue:62]: Yes, would be much more readable.

File: pages/conference/_eventType/index.vue [hunk 3]
```diff
@@ -29,6 +56,9 @@
 import i18n from '@/i18n/conference/speeches.i18n'
 import I18nPageWrapper from '@/components/core/i18n/PageWrapper'
 import CoreH1 from '@/components/core/titles/H1'
+import CoreHr from '@/components/core/layout/Hr.vue'
+import CoreCheckbox from '@/components/core/checkbox/checkbox'
+import CheckboxCollection from '@/components/core/checkbox/checkboxCollection'
```
Comment (mattwang44) [inline pages/conference/_eventType/index.vue:61]:
```suggestion
import CoreCheckbox from '@/components/core/checkbox/Checkbox'
import CheckboxCollection from '@/components/core/checkbox/CheckboxCollection'
```

**Review by mattwang44 — APPROVED** [review]
LGTM. Thanks for making the change so fast!
I'll merge another commit for fixing importing path of checkbox component.

**Pull request merged**